CONTRIBUTING.md

```diff
@@ -55,6 +55,28 @@ Contributions are essential for keeping this extension great. We try to keep it
 
     ![ Launch Extension ](./images/LaunchExtension.png)
 
+### Debugging
+
+After completing the prerequisite steps above, and after launching the **Extension Development Host** for `vscode-xml`, the debugger also extends to `lemminx`, and can be used as follows:
```

Comment:
Instead of "debugger also extends to lemminx", I think that wording it "you can also debug the language server/lemminx" would make it a bit more clear.